script.js

```diff
@@ -1,28 +1,92 @@
+$(document).on("ready", function() {
+
 var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
-  deck: [],
-  players: [],
-  hands: [],
-  buildDeck: function(){
+    deck: [],
+    players: [],
+    hands: [],
+
+    buildDeck: function(){
+        for (i=0; i<values.length; i++) {
+            game.deck.push(values[i]+suits[0]);
```

Comment:
it's a good idea to try and avoid referencing the `global` object whenever possible and take advantage of the object's context